app/assets/javascripts/controllers/container_live_dashboard/container_live_dashboard_controller.js

```diff
@@ -68,11 +70,10 @@ miqHttpInject(angular.module('containerLiveDashboard', ['ui.bootstrap', 'pattern
       $scope.refresh();
     };
 
-    var doRefreshTenant = function (action) {
-      $scope.tenant = action.tenant;
+    $scope.doRefreshTenant = function() {
+      $scope.tenant = $element.find("#ad-hoc-tenant").val();
```

Comment:
So.. what's the advantage of this over reading `$scope.tenant`? If feels *wrong*, since that element has `ng-model="tenant"` and is in the same controller. So the whole line looks like it shouldn't even be needed there.. so why is it?